c2cgeoportal/lib/dbreflection.py

```diff
@@ -70,17 +70,18 @@ class an engine, required for the reflection.
 class _AssociationProxy(object):
     # A specific "association proxy" implementation
 
-    def __init__(self, target, value_attr):
+    def __init__(self, target, value_attr, nullable):
         self.target = target
         self.value_attr = value_attr
+        self.nullable = nullable
 
     def __get__(self, obj, type_=None):
         if obj is None:
             # For "hybrid" descriptors that work both at the instance
             # and class levels we could return an SQL expression here.
             # The code of hybrid_property in SQLAlchemy illustrates
             # how to do that.
-            raise AttributeError  # pragma: no cover
+            return self
```

Comment:
This change looks strange to me ...

Reply:
I get it from sqlalchemy.ext.AssociationProxy class.